app/javascript/controllers/projects/samples/transfers_controller.js

```diff
@@ -16,13 +16,15 @@ export default class extends Controller {
       sessionStorage.getItem(this.storageKeyValue)
     );
 
-    for (const storageValue of storageValues) {
```

Comment:
**nitpick:** This controller should probably be renamed as it is being used by multiple functionality now

Reply:
a35631dccb7ac197f67e8b2843fc9dd990bc4bab